torchdynamo/eval_frame.py

```diff
@@ -364,6 +369,105 @@ def toy_example(a, b):
     )
 
 
+def evaluate(
```

Comment:
I fell like this should live in torchdynamo.testing

Reply:
This isn't used in tests. Why there? Maybe a new dir torchdynamo.tooling?

Reply:
`torchdynamo.testing` has other tools for measuring perf and checking results.  It is not unit test related.